.github/workflows/testcharts.yaml

```diff
@@ -160,10 +153,11 @@ jobs:
         helm install iter8 charts/controller --set logLevel=trace
         kubectl rollout status --watch --timeout=60s statefulset/iter8
 
-    - name: iter8 k launch
+    - name: Start performance experiment
```

Comment:
Performance experiment or performance test? I think it should be the latter, right?